redhat-ipaas-dev-single-tenant.yml

```diff
@@ -365,6 +370,8 @@ objects:
             value: "true"
           - name: JAVA_OPTIONS
             value: "-Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStore=/tls-keystore/openshift-truststore.jks -Duser.home=/tmp"
+            - name: ENDPOINTS_TEST_SUPPORT_ENABLED
```

Comment:
It enables [this endpoint](https://github.com/redhat-ipaas/ipaas-rest/blob/master/rest/src/main/java/com/redhat/ipaas/rest/v1/handler/tests/TestSupportHandler.java) from ipass-rest.